### PR TITLE
chore: Update postinstall

### PR DESCRIPTION
**Pull request by FrankEssenberger**
relates to SAP/cloud-sdk-backlog#485

update the postinstall message once the version 2.0 is released.

### CODE REVIEW

File: packages/core/postinstall.js [hunk 1]
```diff
@@ -10,16 +10,13 @@ function printUpgradeAnnouncement() {
     console.log('🚀 SAP Cloud SDK Announcement');
     console.log('');
     console.log(
-      'The SAP Cloud SDK will release new major versions regularly starting in 2022.'
-    );
-    console.log(
-      'A beta version for version 2 is expected to be released in November 2021.'
+      'The SAP Cloud SDK released version 2.0 on February 2022 and older versions are not actively supported.'
```
Comment (jjtang1985) [inline packages/core/postinstall.js:13]:
```suggestion
      'The SAP Cloud SDK released version 2.0 in February 2022 and older versions are not actively supported.'
```
```suggestion
      'The SAP Cloud SDK released version 2.0 on February 2022 and older versions are not actively supported.'
```

File: packages/core/postinstall.js [hunk 2]
```diff
@@ -10,16 +10,13 @@ function printUpgradeAnnouncement() {
...
     );
     console.log(
       'We will ensure that upgrading takes less than one day of effort when using our upgrade guide.'
     );
     console.log(
-      'Read more in our announcement post: https://sap.github.io/cloud-sdk/docs/js/announcing-version-2'
+      'Read more in our release post: https://blogs.sap.com/2022/02/09/we-proudly-prese…k-for-javascript/'
```
Comment (jjtang1985) [inline packages/core/postinstall.js:19]: The link is broken.

Comment (jjtang1985) [reply packages/core/postinstall.js:19]: This link does not work for me.

Comment (FrankEssenberger) [reply packages/core/postinstall.js:19]: ok, then no preview links work and we have to wait for the publish

**Review by jjtang1985 — APPROVED** [review]
LGTM

**Pull request merged**